samples/sample_decode/src/sample_decode.cpp

```diff
@@ -536,16 +536,16 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
             }
         }
 #endif //MFX_VERSION >= 1022
-        else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-f")))
+        else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-fps")))
```

Comment:
Potential regression due to you remove "-f" key

Reply:
yes, but this option is for humans not for automated script. Human will find out what changed